examples/group.rb

```diff
@@ -35,7 +33,7 @@ def admin?
 puts "Stats for non_admin: #{stats.enabled?(non_admin)}"
 
 puts "\nEnabling Stats for admins...\n\n"
-stats.enable(flipper.group(:admins))
+stats.enable(Flipper.group(:admins))
```

Comment:
Want to change this to `stats.enable_group :admins`? Gets rid of the group method call. I'm fine either way just might be a good time to lean hard into methods at top level instead of feature level.